nuxt/entities/draftLyrics.ts

```diff
@@ -0,0 +1,7 @@
+import { PersistedEntity, TimestampedEntity } from '@/entities/common';
+import { Lyrics } from '@/entities/lyrics';
+
+export interface DraftLyrics extends PersistedEntity, TimestampedEntity {
+  trackId: string;
+  document: Lyrics
```

Comment:
Pretty sure this would use the `LyricsDocument` type I suggested creating above, rather than the whole Lyrics entity, right?

Reply:
Done